src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react"
+import { getUserByUsername } from "../api"
+
+export function ArticleCard ({article}) {
+
+    const [avatarURL, setAvatarURL] = useState("https://www.refugee-action.org.uk/wp-content/uploads/2016/10/anonymous-user.png")
+
+    useEffect(()=>{
+        getUserByUsername(article.author).then(({avatar_url})=>{
+            setAvatarURL(avatar_url)
+        })
+    })
+
+    return (
+    <div id="article-card-container">
+        <h3 id="article-card-title">{article.title}</h3>
+        <div id="article-card-author-img-container">
+        <p id="article-card-author">by {article.author}</p>
+        <img src={avatarURL} alt="author profile picture" id="article-card-author-img"/>
```

Comment:
See what else comes back on the request and potentially include some of that information too. 🙂